### PR TITLE
Add Jest snap files to editorconfig

### PR DESCRIPTION
**Pull request by tpaksu**
This PR provides editorconfig settings for `.snap` files, without this modification, VSCode settings are not enough for preventing the editor to keep trailing whitespace for jest snapshots, which prevents tests from succeeding.

### CODE REVIEW

File: .editorconfig
Comment (luizreis) [inline .editorconfig:28]: Looks like these extra lines slipped through with the commit. Please remove them before merging this PR 😄

**Review by luizreis — APPROVED** [review]
LGTM! Feel free to 🚢 this after the extra blank lines are removed 🚀

File: .editorconfig
Comment (tpaksu) [reply .editorconfig:28]: Sure!

**Pull request merged**
